checkstyle-tester/diff.groovy

```diff
@@ -289,6 +292,38 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateConfigFileHtml(diffDir, configFile) {
+    def configFilenameNoExtension = getFilenameNoExtension(configFile.name)
+    def diffToolDir = Paths.get("").toAbsolutePath()
+        .parent
+        .resolve("patch-diff-report-tool")
+        .toFile()
+    def diffToolJarPath = getPathToDiffToolJar(diffToolDir)
+    this.class.classLoader.rootLoader.addURL(new URL("file:$diffToolJarPath"))
```

Comment:
If we have to print multiple files, it seems like a waste of resources to be loading the JAR multiple times. This HTML report is the main thing under the control of this script doing its life. I can't see us managing another HTML report inside this script except passing more files to the `TextTransform`.

I feel we should move this JAR loading to its own method and always call it when we start the report logic. It will always be called atleast once for the report.

Reply:
This is a good point.

Reply:
Fixed.